### DISCUSSION TITLE
Feesql 扩展中添加 Searchs 会导致异常

### DISCUSSION THREAD

**Discussion by j4587698** [dotnetcore/BootstrapBlazor#3500] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

目前的Freesql扩展使用搜索时会报sql错误。
检查发现在`ToDynamicFilter`方法中仅判断了`if (actions.Filters != null)`
而`SearchFilterAction`中`GetFilterConditions`并没有给`Filters `赋值
就会导致产生一个key和value都为null的条件，freesql转为sql时就会产生一个(())的条件，sql异常

### Expected Behavior

搜索栏输入任意内容
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/24642446/42d82b03-bd36-4b7f-9682-4e3a70b829fc)

此时searches里有一条数据
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/24642446/6841da93-b752-4a5e-a1a6-2b6df3ec296e)

这时候SearchFilterAction中由于没有加Filter，所以actions.Filters是null
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/24642446/48eb2c52-f710-4fd5-a294-4d2630bc1cd4)

这时候item里内容都为null，所以会产生一个(())的条件
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/24642446/0865e68b-ac51-4e78-b0fd-bc583f1c43e0)

**Comment by bb-auto** [comment]
@j4587698 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Reply by ArgoZhang** [reply, marked as answer]
link PR: #3504
